client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/style.scss

```diff
@@ -122,21 +128,33 @@
 }
 
 .sites-dataviews__site-name {
-	display: flex;
-	flex-direction: column;
-	align-items: flex-start;
-	justify-content: flex-start;
+	display: inline-block;
+	text-align: left;
+	text-overflow: ellipsis;
+	overflow: hidden;
 	white-space: nowrap;
+	width: 180px;
 	font-weight: 500;
 	font-size: rem(14px);
 }
 
 .sites-dataviews__site-url {
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
 	font-size: rem(12px);
 	color: var(--studio-gray-60);
 	font-weight: 400;
 }
 
+.preview-hidden {
+	@media (min-width: 2040px) {
```

Comment:
This puzzled me. Why do we have this huge breakpoint here?

Reply:
I have a large screen to test with, and it seemed silly to keep the site name and URL truncated at a certain width when I could increase it for very large screens. But it is a non standard breakpoint, good catch. Maybe not necessary but I figured it would only enhance the display. Then again it's adding additional code to be maintained. So, happy to go either way on this.

Reply:
Thanks for the explanation, seems reasonable!